### PR TITLE
Generate 2023 Annual Report boilerplates

### PR DESCRIPTION
**Pull request by palnabarun**
Generate 2023 Annual Report boilerplates

/committee steering
cc: @kubernetes/steering-committee

### CODE REVIEW

**Review by soltysh — APPROVED** [review]
/lgtm
/hold
for others to chime in

**Comment by pohly** [comment]
Nit: "kubecon"  -> "KubeCon", but let's not bother fixing this.

/lgtm

File: generator/annual-report/sig_report.tmpl [hunk 1]
```diff
@@ -46,52 +46,11 @@
...
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-
-Include any other ways you measure group membership
+<!--
+  Examples include links to email, slides, or recordings.
+-->
```
Comment (mrbobbytables) [inline generator/annual-report/sig_report.tmpl:53]: I think for this we should move it above the generated content, that way its a clear thing between user supplied content and generated content.

File: generator/annual-report/sig_report.tmpl [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+   - Major KEP advancement
+   - Important initiatives that aren't tracked via KEPs
+   - Paying down significant tech debt
+   - Governance and leadership changes
+-->
 
-   -
-   -
-   -
+2.Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
```
Comment (justaugustus) [inline generator/annual-report/sig_report.tmpl:15]:
```suggestion
2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
```

File: generator/annual-report/wg_report.tmpl [hunk 3]
```diff
@@ -1,46 +1,19 @@
...
+<!--
+   Some example items that might be worth highlighting:
+   - artifacts
+   - reports
+   - white papers
+   - work not tracked in KEPs
+-->
 
-Include any other ways you measure group membership
+2.Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
```
Comment (justaugustus) [inline generator/annual-report/wg_report.tmpl:16]:
```suggestion
2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
Quick fix

**Comment by palnabarun** [comment]
@mrbobbytables @BenTheElder @justaugustus -- all comments are resolved. Can you re-LGTM?

**Pull request merged**